### ISSUE TITLE
Support for marking individual items as editable or readonly

### ISSUE THREAD

**Issue by Medha2512** [almende/vis#275]
Hi 
 
I read timeline documentation and it mentioned that individual item editable option would override the entire timeline editable. I have posted sample code at:
http://jsfiddle.net/e7uodse2/68/

Can you help me out as I am still able to edit, delete item 1.

Thanks
Medha

**Comment by AlexDM0** [comment]
Hi,

Over the last year a lot of feature requests have been made. We have just introduced our new website which has a list of the requested features. We have placed this request on that list. 

The list can be found here:
http://visjs.org/featureRequests.html

An explaination of the new system can be found here:
http://visjs.org/blog.html#New\ website\ for\ vis\.js\!

I would like to stress that this does not mean we abandon this request. Discussion here will continue if needed on this feature but we will close it to keep our Github issue page more of a bug-todo list.

Future feature requests will still be made here and then added to the website by us.

Regards,

Alex
